### ISSUE TITLE
Conditional VAE

### ISSUE THREAD

**Issue by chrsunwil** [greenelab/shared-latent-space#22]

**Comment by chrsunwil** [comment]
Use the disease for the condition. Cognoma has a very nice table which will work well for this purpose. Need to decide how to change the project's layout to accommodate this. Should probably be in a different branch

**Comment by chrsunwil** [comment]
This does not seem to help the model. I still have the implementation in a separate branch if I decide to include it in the release.

**Comment by gwaybio** [comment]
> I still have the implementation in a separate branch if I decide to include it in the release.

Yes, please add your implementation! Also, do you think it is worth describing why it doesn't seem to help? Would that take a long time to writeup?

**Comment by chrsunwil** [comment]
Well I'm not exactly sure why it doesn't help the model, but the results are more or less the same as the regular svae, if not a little bit worse at reconstruction. I can spend some more time debugging and trying to make it better, but the scvae still doesn't use the latent space nodes to represent the same differences in the data, which was my primary hope. If I made the conditions features such as mutation of TP53, then it might be able to work better to that end, but that would perhaps become more specific than is helpful? It also wouldn't be clear to me exactly how to pick the conditions.

**Comment by chrsunwil** [comment]
I'll work on the best way of incorporating the scvae into the overall project
